source/beamline/BioXAS/BioXASSideBeamline.cpp

```diff
@@ -208,7 +208,7 @@ void BioXASSideBeamline::setupComponents()
 
 	// Endstation safety shutter.
 
-	endstationShutter_ = new  BioXASEndstationShutter("BioXASSideEndstationShutter", "SSH1607-5-I22-01", this);
+	endstationShutter_ = new  CLSExclusiveStatesControl("SSH1607-5-I22-01", "SSH1607-5-I22-01:state", "SSH1607-5-I22-01:opr:open", "SSH1607-5-I22-01:opr:close", this);
```

Comment:
is it possible to encapsulate  `endstationShutter_` and `frontEndShutters_` into class `BioXASShutters` (as a member of BioXASShutters instead of BioXASBeamline)? 

about the `fastShutter_` below (Ln395), is it a common shutter for main and image? if yes, it might be good to put it in the BioXASShutter too.